apps/ai-image-generator/backend/src/actions/aiig-generate-image.ts

```diff
@@ -57,7 +58,9 @@ export const handler = async (
     numImages: 4,
     size: '1024x1024',
   });
-  const images = openAiImages.filter((image): image is Image => !!image.url);
+  const images = openAiImages
+    .map((image) => ({ url: image.url, imageType: 'png' }))
```

Comment:
Wait - small optimization here. Why can't we filter first and then map. Seems more efficient right? I guess it doesn't really matter with 4 images

Reply:
Honest answer: Harder to deal with types to filter first so I swapped it. The OpenAI type is super annoying to work with since `url` is actually optional.

Some ideas here on how we could just create an interim type to work around it better https://stackoverflow.com/questions/40370214/how-to-override-a-property-to-be-non-nullable-in-typescript